### DISCUSSION TITLE
Alternative port number

### DISCUSSION THREAD

**Discussion by babsey** [nest-desktop/nest-desktop#404] [Ideas] [answered]
By default NEST Desktop (not in the development) runs on port `8000`.

However, the port `8000` is a popular one that other apps also use it.
I purpose a uncommon port for NEST Desktop, e.g. `8532` (reverse Fibonacci).

**Comment by babsey** [comment, marked as answer]
Number should in the range of 49152–65535, which are dynamic ports according to [IANA](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers).

Suggestions:

- [ ] 53211 - reverse Fibonacci
- [x] 54286 - postal code of University of Trier
- [ ] 54321 - descending number from 5
- [ ] 65432 - descending number from 6
- [ ] 65535 - last dynamic number

**Comment by jens-br** [comment]
It has been implemented in #446 .
